src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+    title="Send a custom incident description"
+  >
+  
+  Since we're creating an alert condition that lets us know if there are any latency issues with our `WebPortal` application, we want to make sure our developers have all the information they need when they are notified about this incident. We're going to use workflows to notify a team Slack channel when an incident is created.
+
+  Learn more about custom incident descriptions in our [docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-incident-descriptions).
+
+</Collapser>
+
+  <Collapser
+    id="runbook"
+    title="Add runbook url"
+  >
+If you'd like to link to a runbook, you can put the URL in the runbook URL field.
+  </Collapser>
+</CollapserGroup>
+
+</Step>
+</Steps>
+
+## Edit or improve an existing alert condition [#edit-existing-alert-condition]
+
+If you want to edit an alert condition you've already created, you can:
+
+1. Go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**.
+2. Select **Alert Conditions** in the left navigation.
+3. Click on the alert condition you want to edit.
+
+From there, you will be able to see all of the elements you set when creating your condition. You can edit specific elements of the alert condition by clicking the pencil in top right of each section. 
+
+If you have questions about how to edit each section, refer back to `Create a new alert condition` section:
+
+* [Signal coverage](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition)
+* [Signal behavior](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#advanced-signal-settings)
+* [Thresholds](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#thresholds)
+* [Additional details](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#additional-details) 
+
+### Signal Coverage [#signal-coverage]
+
+Signal coverage gives a closer look at the most recent signals being ingested by the condition. For Synthetics type conditions, a chart is shown with the 5 most recent `SyntheticChecks`. For all other types, a line chart showing the `NraiSignal` over time for the last few days is shown for easier analysis. This chart is similar to what is shown for an individual issue or incident.
+
+If a condition is disabled, you’ll see an empty state message in this section of the Overview. In the case where the condition is not receiving a signal, you will see an empty state prompting you to adjust your condition settings.
+
+#### Troubleshooting your signal history chart
+There are a few reasons that you may see an empty signal history chart.
+- The condition is disabled and not producing any signals.
+- The condition is a NRQL query and the metric or event it is targeting is not instrumenting data.
```

Comment:
Do you mean that the Entity the query targets needs to be instrumented? Agents "instrument" an entity, and the NRQL query exposes the event/metric data captured by the agent, but the query itself doesn't instrument